jobs/candlepinPipelineJob.groovy

```diff
@@ -0,0 +1,40 @@
+import jobLib.rhsmLib
+
+String baseFolder = rhsmLib.candlepinJobFolder
+String githubOrg = binding.variables['CANDLEPIN_JENKINS_GITHUB_ORG'] ?: 'candlepin'
+
+pipeline_helper = job("$baseFolder/candlepin pipeline helper") {
+    logRotator {
+        numToKeep(1)
+    }
+    steps {
+        downstreamParameterized {
+            trigger('candlepin') {
```

Comment:
Yeah, same thing with the subman pipeline. Seems like a bug in the view...

But it doesn't affect the jobs actually running. :smile: